packages/oui-header-tabs/src/header-tabs.controller.js

```diff
@@ -1,20 +1,59 @@
+import _ from "lodash";
```

Comment:
You could only import the `_.prototype.find` function

Reply:
👍